conf/cassandra.yaml

```diff
@@ -1791,3 +1791,4 @@ drop_compact_storage_enabled: false
 #    heartbeat_file: /var/lib/cassandra/data/cassandra-heartbeat
 #    excluded_keyspaces: # comma separated list of keyspaces to exclude from the check
 #    excluded_tables: # comma separated list of keyspace.table pairs to exclude from the check
+
```

Comment:
What's up w/ the newline here?

Reply:
Without it, ant adds it, and dirties the build every time.